app/controllers/dashboard/projects_controller.rb

```diff
@@ -2,29 +2,41 @@ class Dashboard::ProjectsController < Dashboard::BaseController
   inherit_resources
 
   def show
-
+    @project = Project.find(params[:id])
```

Comment:
This controller is using `inherit_resources` so this method should not be necessary since it should already be defined like this

Reply:
Also, this should only show the projects of the user, not all projects.